### ISSUE TITLE
A11y_SQL database projects extension_Advance Publish_Exclude Object Type_ScreenReader : NVDA is announcing the row count wrong when checkbox is checked.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/azuredatastudio#22162]
### GitHub Tags
#A11yWCAG2.1; #A11yTCS; #A11ySev2; #DesktopApp; #ScreenReader; #WCAG1.3.1; #Win11; #SH-SQL Azure Data Studio-Win32-March23; #Win32; #SQL Azure Data Studio; #NVDA; #AILimited; #Info and Relationships;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### Actual Result:​
NVDA is announcing the row count wrong when checkbox is checked.

### Expected Result:​
NVDA should announce the row count correct when checkbox is checked or not checked.

### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.41.2 (user setup)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Screen reader: NVDA

### Repro Steps

1. Turn on NVDA
2. Launch Azure Data Studio
3. Tab till extension and search for “SQL Database Projects” and install it.
4. Tab till database project and press enter.
5. “Database Project” panel will open.
6. Tab till “View And More Action” button and press enter.
7. Tab till “Create Project From Database” and press enter.
8. Tab till server field and press enter.
9. “Connection” dialog will open. Fill the details for SQL server and connect.
10. Tab till Create button and press enter.
11. Database project will get create and will appear in Database project panel.
12. Tab till created project name in Database Project panel.
13. Press context menu button(right click).
14. Tab till Publish and press enter.
15. Publish Project window will open.
16. Tab till “Advance” button and press enter.
17. Tab till Exclude Object Type tab and press enter.
18. Verify whether the NVDA is announcing the row count correct when checkbox is checked or not.

### User Impact:
Screen reader users will not know on which row they are and get confused about the row count.

### Attachment:

![Presentation4](https://user-images.githubusercontent.com/124244478/223093379-823854ac-b6c8-410e-ad22-700dc9c66ff1.gif)

**Comment by kisantia** [comment]
This seems like screenreader issue with NVDA: https://github.com/nvaccess/nvda/issues/14332

The example in the linked issue also has the same problem of incorrect row being announced if you scroll down the table, then go back up to the first row the incorrect row is announced.
![incorrectRowNvda](https://user-images.githubusercontent.com/31145923/226064437-16146ee1-7cf0-46d4-ba6c-9cb6f8203fb3.gif)

**Comment by kisantia** [comment]
@AnKushSingh05 this looks like a screenreader issue with NVDA, as mentioned in the comment above.

**Comment by AnKushSingh05** [comment]
@kisantia I have checked with Jaws and Narrator both the screen readers issue is repro'ing with both the screen reader .

**Comment by kisantia** [comment]
@AnKushSingh05 this has been fixed in the latest version of the sql database projects extension (0.23.0). Please download the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch) and verify this is fixed. Thanks!

**Comment by kisantia** [comment]
@AnKushSingh05 please verify this is fixed when you have a chance

**Comment by AnKushSingh05** [comment]
#Closed
Regressed the issue in the below environment
Version: 1.43.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/afafee844ce2b07eb55bdb6f40ee1ce88c0bbf4e
Date: 2023-03-29T05:28:54.263Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Issue is fixed. Hence closing the bug.
![Bug_22162_Fixed](https://user-images.githubusercontent.com/124244478/229018640-a6d95bd8-8c71-4ff7-ae60-506b1f74da99.gif)
